PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/LaundrySettingsAdapter.java

```diff
@@ -32,15 +32,15 @@
  */
 
 public class LaundrySettingsAdapter extends BaseExpandableListAdapter {
-    private List<String> laundryHalls;
-    private HashMap<String, List<LaundryRoomSimple>> laundryRooms;
-    private Context mContext;
-    private SharedPreferences sp;
-    private String s;
-    private List<Switch> switches = new ArrayList<>();
-    private int maxNumRooms = 3;
+    private final List<String> laundryHalls;
+    private final HashMap<String, List<LaundryRoomSimple>> laundryRooms;
+    private final Context mContext;
+    private final SharedPreferences sp;
+    private final String s;
+    private final List<Switch> switches = new ArrayList<>();
+    private final int maxNumRooms = 3;
     private StudentLife labs;
-    private String bearerToken;
+    private final String bearerToken;
```

Comment:
is this just to clean up some other code? Just was wondering. Also, side note but weird that this is in Java (along with the other Laundry and Menu adapter)

Reply:
I honestly don't remember doing this at all. All the java files are old stuff that I haven't touched and didn't need to interact with while building PCA. Will revert to the old one in a commit right now.